airbyte/cloud/connections.py

```
@@ -112,6 +120,14 @@
         )
         return self._cloud_destination_object
 
+    @property
+    def destination(self) -> CloudDestination:
```

Ruff (0.8.0)

Redefinition of unused `destination` from line 112

(F811)